contents/docs/feature-flags/best-practices.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Feature flags best practices
+sidebar: Docs
+showTitle: true
+---
+## 1. Call your flag in as a few places as possible
+
+It should be easy to understand how feature flags affect your code. The more locations a flag is, the more likely it is to cause problems. For example, a developer could remove the flag in one place but forget to remove it in another.
+
+If you expect to use a feature flag in multiple places, it's a good idea to wrap the flag in a single function or method. For example:
+
+```js
+function useBetaFeature() {
+    return posthog.isFeatureEnabled('beta-feature')
+}
+```
+
+## 2. Identify users
+
+Because PostHog evaluates flags based on the user's distinct ID, having different IDs can cause the same user to receive different flag values across different sessions, devices, and platforms. By [identifying](/docs/getting-started/identify-users) them, you can ensure they receive the same flag value.
+
+## 3. Use server-side local evaluation for faster flags
+
+Evaluating feature flags requires making a request to PostHog for each flag. However, you can improve performance by evaluating flags locally. Instead of making a request for each flag, PostHog will periodically request and store feature flag definitions locally, enabling you to evaluate flags without making additional requests.
+
+Evaluating flags locally when possible, since this enables you to resolve flags faster and with fewer API calls. See our docs on [local evaluation](/docs/feature-flags/local-evaluation) for more details.
```

Comment:
```suggestion
Evaluate flags locally when possible, since this enables you to resolve flags faster and with fewer API calls. See our docs on [local evaluation](/docs/feature-flags/local-evaluation) for more details.
```